src/Uno.UI/UI/Xaml/Media/LinearGradientBrush.cs

```diff
@@ -14,6 +16,9 @@ public LinearGradientBrush()
 			double angle)
 		{
 			GradientStops = gradientStopCollection;
+
+			var rad = MathEx.ToRadians(angle);
+			EndPoint = new Point(Math.Cos(rad), Math.Sin(rad));
```

Comment:
This is not important at all, it's almost impossible for a real production code to rely on that behavior, but:

In UWP, `new LinearGradientBrush(new GradientStopCollection(), 45).EndPoint.X;` and `new LinearGradientBrush(new GradientStopCollection(), 45 + 360).EndPoint.X;` are EXACTLY equal.

However, `Math.Sin` and `Math.Cos` will return **almost** (but not exactly) equal results.

Consider doing `% 360` and add a test for exact equality.

Reply:
I tried this snippet in UWP:

```
var linear = new LinearGradientBrush(new GradientStopCollection(), 3287);
var linear2 = new LinearGradientBrush(new GradientStopCollection(), 3647);
Assert.AreEqual(linear.EndPoint.X, linear2.EndPoint.X);
Assert.AreEqual(linear.EndPoint.Y, linear2.EndPoint.Y);
```

But the first assert failed - with some little decimal number different. So I guess they also just convert and apply `Cos`/`Sin` 🤔

Reply:
@MartinZikmund Interesting. I confirm I see the behavior you described for UWP, but didn't happen for 45 degree. Let's ignore this unimportant difference.